engine/spverrors/definitions.go

```diff
@@ -375,3 +375,23 @@ var ErrBroadcastWrongBUMPResponse = models.SPVError{Message: "broadcast server r
 
 // ErrBroadcastRejectedTransaction is when broadcast server rejected transaction
 var ErrBroadcastRejectedTransaction = models.SPVError{Message: "broadcast rejected transaction", StatusCode: 400, Code: "error-broadcast-rejected-transaction"}
+
+// ////////////////////////////////// BLOCK HEADER SERVICE ERRORS
```

Comment:
I would suggest to move this section to separate file in package actions/merkleroots/errors